projects/angular/src/popover/signpost/_variables.signpost.scss

```diff
@@ -17,4 +17,4 @@ $clr-signpost-border-radius: variables.$clr-global-borderradius !default;
 
 $clr-signpost-pointer-border: $clr-signpost-border-size solid $clr-signpost-content-border-color !default;
 $clr-signpost-pointer-invisible-border: $clr-signpost-border-size solid transparent !default;
-$clr-signpost-pointer-psuedo-border: $clr-signpost-border-size solid $clr-signpost-content-bg-color !default;
+$clr-signpost-pointer-pseudo-border: $clr-signpost-border-size solid $clr-signpost-content-bg-color !default;
```

Comment:
This is a breaking rename to an existing SCSS variable. Since it would break products even without the shim being applied, we should not include this change in a non-major release. I would file an issue to rename this in a breaking change PR for v16.

This applies to the CSS custom property rename as well.

Reply:
I'll revert it, but we should take car of it in the next release.

Reply:
corrected in commit 9677cdc